src/sections/Community/Newcomers-guide/index.js

```diff
@@ -10,64 +10,121 @@ import CommunityImage4 from "./community-image4.png";
 import TutorialsTable from "./Tutorials-table";
 import NewcomersMap from "./newcomers-map";
 import JoinCommunity from "../Join-community";
-
+import longArrow from "./long-arrow.svg";
 
 const NewcomersGuide = () => {
+  const [hover, setHover] = useState(false);
+
   return (
     <NewcomersGuideWrapper>
-      <PageHeader title="Welcome Newcomers!"/>
+      <PageHeader title="Welcome Newcomers!" />
       <Container>
         <div className="newcomers-welcome">
           <p>
-            Whether you are a first time contributor or an open source veteran,you are welcome to
-            contribute to and actively engage in the development of projects at Layer5.
-            Here are some instructions to get you started and if you haven’t joined yet,
-            join the <a href="http://slack.layer5.io/" target="_blank" rel="noreferrer">Slack workspace</a> to collaborate with the community.
-            Also, you can check the quick-links below for jumping straight into things.
+            Whether you are a first time contributor or an open source
+            veteran,you are welcome to contribute to and actively engage in the
+            development of projects at Layer5. Here are some instructions to get
+            you started and if you haven’t joined yet, join the{" "}
+            <a href="http://slack.layer5.io/" target="_blank" rel="noreferrer">
+              Slack workspace
+            </a>{" "}
+            to collaborate with the community. Also, you can check the
+            quick-links below for jumping straight into things.
           </p>
           <Row className="newcomers-journey">
-            <Col className="heading" sm={12} lg={6}>
+            <Col
+              className="heading"
+              sm={12}
+              lg={6}
+              onMouseOver={() => setHover({ hover: true })}
```

Comment:
Even these can also be changed now right?

Reply:
I'll update that as well